mapper/mapper_test.go

```diff
@@ -452,6 +457,8 @@ func TestUnitCensus(t *testing.T) {
 			t.Error("failed to get config")
 		}
 
+		censusDataURLQuery := getCensusDataURLQuery(availableItems)
```

Comment:
I would not call the code here as we wouldn't be checking it behaves as expected. We should just hardcode the expected result

```suggestion
		expectedCensusDataURLQuery := "?topics=1234,4567&filter=datasets"
```